src/view/screens/Settings/index.tsx

```diff
@@ -531,8 +531,8 @@ export function SettingsScreen({}: Props) {
           ]}
           onPress={openFollowingFeedPreferences}
           accessibilityRole="button"
-          accessibilityHint=""
-          accessibilityLabel={_(msg`Opens the home feed preferences`)}>
+          accessibilityLabel={_(msg`Home feed preferences`)}
```

Comment:
As a result of #2965 `Home` to `Following` should be updated here:

```suggestion
          accessibilityLabel={_(msg`Following feed preferences`)}
```